lib/phlex/view.rb

```diff
@@ -9,6 +9,16 @@ class View
 		extend HTML
 		include Renderable
 
+		module TemplateOverride
+			def template(**kwargs, &block)
+				if block.instance_variable_get(:@rendering)
```

Comment:
If the block has the `@rendering` instance variable, we know this was called from `#call` and the intention is to render the view.